flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamMockEnvironment.java

```diff
@@ -431,4 +432,9 @@ public void setCheckpointResponder(CheckpointResponder checkpointResponder) {
     public ChannelStateWriteRequestExecutorFactory getChannelStateExecutorFactory() {
         return channelStateExecutorFactory;
     }
+
+    @Override
+    public CheckpointExpiredThreadDumper getCheckpointExpiredThreadDumper() {
+        return null;
```

Comment:
I prefer to use `Optional` here due to Flink's [code-style](https://flink.apache.org/how-to-contribute/code-style-and-quality-java/#java-optional)

Reply:
I removed all the explicit null value return.